pkg/roachprod/install/cockroach.go

```diff
@@ -1561,3 +1561,16 @@ func getEnvVars() []string {
 	}
 	return sl
 }
+
+// SuppressMetamorphicConstantsEnvVar returns the env var to disable metamorphic testing.
+// This doesn't actually disable metamorphic constants for a test **unless** it is used
+// when starting the cluster. This does however suppress the metamorphic constants being
+// logged for every cockroach invocation, which while benign, can be very confusing as the
+// constants will be different than what the cockroach cluster is actually using.
+//
+// TODO(darrylwong): Ideally, the metamorphic constants framework would be smarter and only
+// log constants when asked for instead of unconditionally on init(). That way we can remove
+// this workaround and just log the constants once when the cluster is started.
+func SuppressMetamorphicConstantsEnvVar() string {
+	return "COCKROACH_INTERNAL_DISABLE_METAMORPHIC_TESTING=true"
```

Comment:
Nit: maybe this could be a constant in `config`, where it's already used in the list of defaults?

Reply:
Done